rust/item/src/lib.rs

```diff
@@ -2638,7 +2638,7 @@ pub struct ItemClassJoinNamespace<'info> {
     #[account()]
     namespace: UncheckedAccount<'info>,
     #[account(address = anchor_lang::solana_program::sysvar::instructions::ID)]
-    pub instructions: UncheckedAccount<'info>,
+    instructions: UncheckedAccount<'info>,
```

Comment:
Why no more pub for instructions

Reply:
to match the other ix styling

Reply:
They should all be pub so other rust crates can create/use these type. Or we need to have a constructor to build them.

Reply:
Every other account struct in all our programs do not use the `pub`, so this would be a pretty huge change (not saying its a bad idea though)

Reply:
Let's leave this one as pub in that case and we can update the rest in a later PR